src/aks-preview/HISTORY.rst

```diff
@@ -16,6 +16,10 @@ Pending
 * Raise a ClientRequestError when creating the same cluster again in command `az aks create`.
 * Vendor new SDK and bump API version to 2023-02-02-preview.
 
+0.5.133
++++++++
+* Add cluster upgrade settings options `--enable-upgrade-ignore-kubernetes-deprecations`, `--disable-upgrade-ignore-kubernetes-deprecations`, and `--upgrade_override_until`.
```

Comment:
Please include the items under the pending section as well

Reply:
Also there's another PR #6082 trying to do a release, will have conflict

Reply:
@wenjungaogaogao  --enable-upgrade-ignore-kubernetes-deprecations is maybe too long and probably going to exceed the char count limit for parameters?

Reply:
We already have a few such args that are longer than the suggested length. This should not be a param that cx use often and we'd prefer them not to set it. So leaving it a as self explanatory.